src/index.ts

```diff
@@ -15,3 +16,17 @@ const pluginName = 'audioPlayer';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
 registerPlugin(pluginName, AudioPlayer);
+
+
+// @ts-ignore
+window.__kalturaplayerdata = {
```

Comment:
this will override the existing uiConf from the server.
if there's multiple players on the page, it will override this for all of the players.
also, this will couple audio player to the player. if the player changes the way it works with ui conf configuration, it will affect this code.

in short, i think we should look at another way to do this.

Reply:
if there are multiple players, it is also true for the BE variable... think of it...